app/Http/Controllers/Serivice/ServiceController.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use App\Http\Controllers\Service\ServiceRepository;
+use App\Service;
+use app\Http\Requests\CreateServiceRequest;
+
+class ServiceController extends Controller
+{
+    private $serviceRepository;
+
+    public function __construct(ServiceRepository $serviceRepository)
+    {
+        $this->serviceRepository = $serviceRepository;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        // TO-DO
+        // get the pagination number or a default to 10
+        $services = $this->$serviceRepository->paginate(config('settings.pagination.small')) ?? 10;
```

Comment:
See my previous comment on removing the value `10`.

Reply:
Have done it